python/py-fire/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-fire
+version             0.3.1
+platforms           darwin
```

Comment:
if this port doesn't compile anything (please check!), then add supported_archs noarch

Reply:
Thanks done.